editor/import/resource_importer_scene.cpp

```diff
@@ -180,6 +180,17 @@ bool ResourceImporterScene::get_option_visibility(const String &p_option, const
 		return false;
 	}
 
+	// Global settings
```

Comment:
Global like global to Godot Engine?

Reply:
Okay, I agree, these are poorly explained comments. I'll update this with better clarity.